examples/js/controls/OrbitControls.js

```diff
@@ -514,20 +517,48 @@
 
 			}
 
-			function handleMouseWheel( event ) {
-
+			function followZoom(event){
+				let x = (event.offsetX / scope.domElement.clientWidth) * 2 -1,
+				y = -(event.offsetY / scope.domElement.clientHeight) * 2 + 1,
+				v = new Vector3(x, y, 0);
```

Comment:
Okay, it's because the `THREE` namespace is missing.

Reply:
Ummm better not to edit `examples/js` by hand.. let the script do it

Reply:
PRs should contain only the `examples/jsm` code.

Reply:
@jiannanya You do that by just editing `examples/jsm` files and then execute `npm run build-examples`.

Reply:
Sorry for that.
I have run the command `npm run build-examples`  but got hundreds of JS files changed.
Is it normal?  
![files changed](https://raw.githubusercontent.com/jiannanya/image_host/main/threejs_npm_.png)

Reply:
My node version is v16.13.2 and my npm version is 8.1.2

Reply:
Well, just discard these changes only keep the updated `OrbitControls` file.